uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/SicknessRequest.php

```diff
@@ -169,8 +169,16 @@ function civicrm_api3_sickness_request_isvalid($params) {
   catch (CRM_HRLeaveAndAbsences_Exception_InvalidSicknessRequestException $e) {
     $result[$e->getField()] = [$e->getExceptionCode()];
   }
+  catch (CRM_HRLeaveAndAbsences_Exception_InvalidLeaveRequestException $e) {
```

Comment:
Since both exception extend `CRM_HRLeaveAndAbsences_Exception_EntityValidationException`, you only need one catch:

```php
   try {
     CRM_HRLeaveAndAbsences_BAO_SicknessRequest::validateParams($params);
   }
    catch (CRM_HRLeaveAndAbsences_Exception_EntityValidationException $e) {
      $result[$e->getField()] = [$e->getExceptionCode()];
    }
```